tests/components/LocationBias.test.tsx

```diff
@@ -0,0 +1,148 @@
+  location: {
+    locationBias: {
+      latitude: null,
+      longitude: null,
+      displayName: null, 
+      method: null
+    }
+  },
+}
+
+const newGeoPosition = {
+  coords: {
+    accuracy: null,
+    altitude: null,
+    altitudeAccuracy: null,
+    heading: null,
+    latitude: 40.741591687843005,
+    longitude: -74.00530254443494,
+    speed: null,
+  },
+timestamp: null
+};
+
+describe("LocationBias", () => {
+  beforeEach(() => {
+    mockAnswersHooks({
+      mockedState: mockedStateVaDevice,
+      mockedActions: {
+        state: mockedStateVaDevice,
+        setUserLocation: jest.fn(),
+      } 
+    });
+    jest.spyOn(locationOperations, "getUserLocation").mockImplementation(async ()=> {
+      return Promise.resolve(newGeoPosition);
+    });
+    jest.spyOn(searchOperations, "executeSearch").mockImplementation();
+    // jest.spyOn(searchOperations, "executeSearch").mockImplementation(()=> new Promise(()=>null));
+  });
+
+  it("Proper text is rendered (Location name, method, and update btn)", () => {
+    render(<LocationBias />);
+    const expectedLocationName = mockedStateVaDevice.location.locationBias.displayName;
+    const locationNameElement = screen.getByText(expectedLocationName);
+    expect(locationNameElement).toBeDefined();
+
+    const expectedMethodMessage = "(based on your device) -";
+    const basedOnMethodElement = screen.getByText(expectedMethodMessage);
+    expect(basedOnMethodElement).toBeDefined();
+
+    const updateLocationButton = screen.getByText("Update your location");
+    expect(updateLocationButton).toBeDefined();
+  })
+
+  it("On update location click, setUserLocation is called with params being coordinates returned by getUserLocation", async () => {
+    const actions = spyOnActions();
+    render(<LocationBias />)
+    clickUpdateLocation();
+
+    const expectedCoordinates = {
+      latitude: newGeoPosition.coords.latitude,
+      longitude: newGeoPosition.coords.longitude
+    }
+    
+    await waitFor(() => {
+      expect(locationOperations.getUserLocation).toBeCalled();
+      expect(actions.setUserLocation).toBeCalledWith(expectedCoordinates);
+    })
+  });
+
+  it('On location change, clicking "Update your location" updates rendered location name', async () => {
+    render(<LocationBias />);
+    const expectedLocationName = mockedStateVaDevice.location.locationBias.displayName; 
+    const locationNameElement = screen.getByText(expectedLocationName);
+    expect(locationNameElement).toBeDefined();
+
+    mockAnswersState(mockedStateNyIP);
+    clickUpdateLocation();
+
+    await waitFor(() => {
+      expect(searchOperations.executeSearch).toBeCalled();
+    })
+
+    const newExpectedLocationName = mockedStateNyIP.location.locationBias.displayName; 
+    const newLocationNameElement = screen.getByText(newExpectedLocationName);
+    expect(newLocationNameElement).toBeDefined();
+  });
+});
+
+it('Correct device attribution message rendered, device', () => {
```

Comment:
I would either include these tests in the big describe('LocationBias') block or just remove the describe block. I prefer to not have the describe block, especially for a component test file which is pretty explanatory, but most of our other component tests have this block that somehow I've missed my chance to get on a soapbox for so up to you which you prefer

Reply:
removed describe block